### DISCUSSION TITLE
Is there anyway to scale?

### DISCUSSION THREAD

**Discussion by demonguy** [oracle/opengrok#4406] [General]
We have multiple Big AOSP repostitories.
Now it's very large that we need multiple machines to hold them.
Is there anyway to use same domain for these different machines?

**Comment by vladak** [comment]
> We have multiple Big AOSP repostitories.
> Now it's very large that we need multiple machines to hold them.
> Is there anyway to use same domain for these different machines?

What is the limit you are bumping into ? Indexing time, memory constraint, storage space or something else ?

**Reply by demonguy** [reply]
Disk limitation.
We are now holding 50+ different AOSP variant for our different products. The total size is more than 50TB+. So we have to depoly them on different machines which have different domain name for users. It's a little bit tricky. What we want is to deploy repos on different machine, but maintain the same domain and url for users

**Comment by demonguy** [comment]
Is there any exist solution for it?
